app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    source_file_name = command_split[1]
+    directory = "/".join(command_split[2].split("/")[:-1])
+    destination_file_name = command_split[2].split("/")[-1]
+
+    print(source_file_name)
+    print(directory)
+    print(destination_file_name)
+
+    if directory:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+        with open(source_file_name, "r") as file, open(directory + "/" + destination_file_name, "w") as d_file:
+            d_file.write(file.read())
+        os.remove(source_file_name)
```

Comment:
The `os.remove(source_file_name)` line will delete the source file after copying its contents to the destination. Ensure that the file is successfully written before removing the source file to prevent data loss.